<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>588</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>588</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>bibliothèque</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/images/images/biblio.png</normaloff>:/new/images/images/biblio.png</iconset>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="list_book_display">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>0</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>titre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>auteur</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> l'année du publication</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>loueur</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="text_book">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>371</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_book_but">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/images/images/settring.png</normaloff>:/new/images/images/settring.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="list_bib_display">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>0</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>nom de la bibliothèque</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nombre des livres</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_bib_but">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_book_but">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_book_but">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_bib_but">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/images/images/refresh.png</normaloff>:/new/images/images/refresh.png</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+R</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
    width: 22px; /* Set width of the indicator */
    height: 22px; /* Set height of the indicator */
    border: solid;
    background-color: lightgray;
}
QCheckBox::indicator:unchecked {
    image: url(:/new/images/images/search_n.png);
}
QCheckBox::indicator:checked {
    image: url(:/new/images/images/search_p.png);
}
QCheckBox{
    background-color: lightgray;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_rent">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>louer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_bib_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_book_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>sauvegarder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/images/images/settring.png</normaloff>:/new/images/images/settring.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_search">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>la méthode de la recherche : auther</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionadder">
   <property name="text">
    <string>ajouter un livre</string>
   </property>
  </action>
  <action name="actionsupp_le_livre_selecter">
   <property name="text">
    <string>supp le livre selecté</string>
   </property>
  </action>
  <action name="actionselectionner_le_bibilo">
   <property name="text">
    <string>selectionner le bibilo</string>
   </property>
  </action>
  <action name="actionajouter_une_biblio">
   <property name="text">
    <string>ajouter une biblio</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
